core/chains/evm/config/chain_specific_config.go

```diff
@@ -226,7 +226,6 @@ func setChainSpecificConfigDefaultSets() {
 	polygonMainnet.finalityDepth = 500           // It is quite common to see re-orgs on polygon go several hundred blocks deep. See: https://polygonscan.com/blocks_forked
 	polygonMainnet.gasBumpThreshold = 5          // 10s delay since feeds update every minute in volatile situations
 	polygonMainnet.gasBumpWei = *assets.GWei(20)
-	polygonMainnet.gasPriceDefault = *assets.GWei(1)
```

Comment:
Why was this removed?

Reply:
Just conflicts that auto-resolved - I'll clean it up when I update the defaults again.

Reply:
Oh I misremembered. This one is actually redundant, since six lines later we do:
```go
polygonMainnet.gasPriceDefault = *assets.GWei(30)
```
Probably just overlooked when it was raised, or revived from an old merge conflict :shrug: